### PR TITLE
feat(driver): add support for PagerDuty

### PR DESCRIPTION
**Pull request by MrWazaby**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -1,161 +1,202 @@
...
+		"payload": map[string]string{
+			"summary":  message,
+			"source":   "isThisUp",
+			"severity": "critical",
+		},
+	}
+	jsonValue, _ := json.Marshal(values)
+
+	client := &http.Client{}
+	req, _ := http.NewRequest("POST", "https://events.pagerduty.com/v2/enqueue", bytes.NewBuffer(jsonValue))
```
Comment (lauevrar77) [inline main.go:87]: Manage request creation error ?

File: main.go [hunk 2]
```diff
@@ -1,161 +1,202 @@
...
-        }
-        
-        time.Sleep(time.Duration(sleepingTime) * time.Second)
-    }
+	websiteUrl, haveValue := os.LookupEnv("URL")
+	if !haveValue {
+		log.Fatalln("No URL env variable. Quitting...")
+	}
+
+	plateforme, haveValue := os.LookupEnv("PLATEFORME")
```
Comment (lauevrar77) [inline main.go:122]: PLATFORM in English ?

File: main.go [hunk 3]
```diff
@@ -1,161 +1,202 @@
...
+	if !haveValue {
+		log.Fatalln("No PLATEFORME env variable. Quitting...")
+	}
+
+	if plateforme != "pagerduty" && plateforme != "opsgenie" {
```
Comment (lauevrar77) [inline main.go:127]: platform in English ?

**Review by lauevrar77 — COMMENTED** [review]
We should look at formatting. Yours seems to have 5 spaces and mine 4.

**Comment by MrWazaby** [comment]
> We should look at formatting. Yours seems to have 5 spaces and mine 4.

I also use 4 spaces but `go fmt` format my spaces as tabs (before running `go fmt` I have the same formatting than you).

**Pull request merged**